ethereum/core/src/test/java/org/hyperledger/besu/ethereum/privacy/MultiTenancyPrivacyControllerTest.java

```diff
@@ -418,4 +419,26 @@ public void getContractCodeFailsForInvalidEnclaveKey() {
                     PRIVACY_GROUP_ID, Address.ZERO, Hash.ZERO, ENCLAVE_PUBLIC_KEY2))
         .hasMessage("Privacy group must contain the enclave public key");
   }
+
+  @Test
+  public void verifyPrivacyGroupMatchesEnclaveKeySucceeds() {
+    final PrivacyGroup privacyGroup =
+        new PrivacyGroup(PRIVACY_GROUP_ID, Type.PANTHEON, "", "", List.of(ENCLAVE_PUBLIC_KEY1));
```

Comment:
Could create a const privacy group ... The same one is used twice.

Reply:
Done :)